src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -752,3 +752,7 @@ fluent-switch.table-switch::part(label) {
     max-height: 400px;
     overflow-y: auto;
 }
+
+.resource-icon {
+    vertical-align: text-bottom;
```

Comment:
Use .align-text-bottom unless you are planning on adding additional styling

Reply:
I think we need to add space to the right as well as per the other comments.